proxy/m2_actions.c

```diff
@@ -1886,6 +1886,77 @@ enum http_rc_e action_content_delete (struct req_args_s *args) {
 	return _reply_m2_error (args, err);
 }
 
+static enum http_rc_e
+_m2_content_delete_many (struct req_args_s *args, struct json_object * jbody) {
+	guint max_request = OIO_CONTENT_DELETE_MANY_NB_MAX_REQ;
+	GError *err = NULL;
+	const char *url_path = oio_url_get(args->url, OIOURL_PATH);
+	json_object *jarray = NULL;
+	PACKER_VOID(_pack) { return m2v2_remote_pack_DEL (args->url); }
+
+	if (!oio_url_get(args->url, OIOURL_ACCOUNT)
```

Comment:
You can consider using `oio_url_has_fq_container()` to check that all the container's fields are present